src/ioctls/vm.rs

```diff
@@ -530,7 +531,7 @@ impl VmFd {
     ///
     /// # Arguments
     ///
-    /// * `fd` - FD which will be signaled. When signaling, the usual `vmexit` to userspace
+    /// * `fd` - `EventFd` which will be signaled. When signaling, the usual `vmexit` to userspace
```

Comment:
nit: maybe use an explicit variable name
```suggestion
    /// * `evtfd` - `EventFd` which will be signaled. When signaling, the usual `vmexit` to userspace
```